@core/cms/src/external-auth-integration/auth-providers/basic-auth/mixpanel/v1/mixpanel.v1.config.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from "@nestjs/common";
+import { BasicAuthConfig } from "../../@types";
+
+/**
+ * Service for configuring Mixpanel V1 Basic HTTP Authentication.
+ */
+@Injectable()
+export class MixpanelV1BasicAuthConfig {
+  public basicAuthConfig: BasicAuthConfig;
+
+  constructor() {
+    this.basicAuthConfig = {
+      provider: {
+        authenticateUrl: "https://mixpanel.com/api/app/me",
+        verifyUrl: "https://mixpanel.com/api/app/organizations/{organizationId}/service-accounts",
```

Comment:
Could you explain what is organizationId? How can I get this? If we host this as CMS, is this organizationId provided by client?